apps/web/src/components/VerificationCode/MethodButton.tsx

```diff
@@ -0,0 +1,60 @@
+import { IUser, Void } from "@litespace/types";
+import { useFormatMessage } from "@litespace/ui/hooks/intl";
+import { LocalId } from "@litespace/ui/locales";
+import { Typography } from "@litespace/ui/Typography";
+import cn from "classnames";
+
+type MethodButtonProps = {
+  method: IUser.NotificationMethodLiteral;
+  isActive?: boolean;
+  onClick?: Void;
+  activeColor: string;
+  icon: React.ReactNode;
+};
+
+const NOTFICATION_METHOD_TO_INTL_MSG_ID: Record<
+  IUser.NotificationMethodLiteral,
+  LocalId
+> = {
+  telegram: "notification-method.telegram",
+  whatsapp: "notification-method.whatsapp",
+};
+
+export function MethodButton({
+  isActive,
+  icon,
+  method,
+  activeColor,
+  onClick,
+}: MethodButtonProps) {
+  const intl = useFormatMessage();
+
+  return (
+    <button
+      onClick={onClick}
+      type="button"
+      className={cn(
+        "relative grow overflow-hidden rounded-2xl transition-colors duration-200 bg-natural-200",
+        "focus-visible:ring-secondary-500 focus-visible:ring-offset-2 focus-visible:outline-none focus-visible:ring-2",
+        isActive && "hover:bg-natural-300"
```

Comment:
The hover effect doesn't work. 


```suggestion
        !isActive && "hover:bg-natural-300"
```